.circleci/config.yml

```diff
@@ -0,0 +1,86 @@
+version: 2.1
```

Comment:
Just a general question: When do we use circle and when github actions? Can every project choose? Because I guess we pay for circle and not for gh actions in public repos.

Reply:
The ultimate goal is to migrate all projects to circleci. I can provide more info in private as this is a public repo